project-base/templates/Front/Content/Category/panel.html.twig

```diff
@@ -15,7 +15,7 @@
                 <a href="{{ url('front_product_list', { id: categoryWithLazyLoadedVisibleChildren.category.id }) }}" class="list-menu__item__link list-menu__item__link--level-{{ categoryWithLazyLoadedVisibleChildren.category.level }} {% if isCurrentCategory %}current{% endif %}">
                     {{ categoryWithLazyLoadedVisibleChildren.category.name }}
                     {% if categoryWithLazyLoadedVisibleChildren.hasChildren %}
-                        <i class="list-menu__item__control svg svg-arrow js-category-collapse-control" data-url="{{ url('front_category_branch', { parentCategoryId: categoryWithLazyLoadedVisibleChildren.category.id }) }}"></i>
+                        {{ icon('arrow', { class: 'list-menu__item__control js-category-collapse-control', data: { url: url('front_category_branch', { parentCategoryId: categoryWithLazyLoadedVisibleChildren.category.id }) } } ) }}
```

Comment:
This line is hard to read. Cannot we try to apply any better formating of it - it would be lovely? (it applies also to others ocurences)

Reply:
I have updated long lines

Reply:
It is much better! Thanks 🎉